File: app/src/main/java/com/example/mvpcalculator/viewmodel/Quoteviewmodel.java
package com.example.mvpcalculator.viewmodel;

class QuoteViewModel : ViewModel() {
        val quoteModel = MutableLiveData<QuoteModel>()
        fun randomQuote() {
        val currentQuote = QuoteProvider.random()
        quoteModel.postValue(currentQuote)
        }
}

File: app/src/main/java/com/example/mvpcalculator/viewmodel/QuoteViewmodel.kt
package com.example.mvpcalculator.viewmodel

import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

    class QuoteViewModel : ViewModel() {
        val quoteModel = MutableLiveData<QuoteModel>()
        fun randomQuote() {
            val currentQuote = QuoteProvider.random()
            quoteModel.postValue(currentQuote)
        }
    }

}